game/CharacterSheet/CharacterSheet.gd

```diff
@@ -169,11 +164,10 @@ func _on_CloseButton_button_up():
 
 func _on_EthnicityStep_ethnicity_chosen(ethnicity):
 	self.character_stats_element.ethnicity = ethnicity["ethnicity_name"]
-	DatabaseOperations.db_update_player_ethnicity(ethnicity["ethnicity_name"])
 	
 
 func _format_ethnicity_trait_name(trait_button):
-	if trait_button.trait_name == "Wszechstronność do kwadratu":
+	if trait_button.trait_name == DatabaseOperations.read_versatility_name_for_button():
```

Comment:
Now that i think about it, it is not a best way to do it. We have to discuss cleaner solution.

Reply:
You fixed that :)